tsdb/chunkenc/chunk_test.go

```diff
@@ -110,24 +110,23 @@ func testChunk(t *testing.T, c Chunk) {
 }
 
 func benchmarkIterator(b *testing.B, newChunk func() Chunk) {
+	const samplesPerChunk = 250
```

Comment:
Why not taking 120 as it's the usual value?

Reply:
The number 250 comes from this line: https://github.com/prometheus/prometheus/pull/5983/files#diff-37dd1fb1ccb424c10039dd47e623d671L136

Probably I should remove that line since it doesn't do anything now.

Possibly I should replace the other `250` in the file.